sympy/ntheory/modular.py

```diff
@@ -54,17 +54,7 @@ def crt(m, v, symmetric=False, check=True):
        >>> [639985 % m for m in [99, 97, 95]]
        [49, 76, 65]
 
-    If the moduli are not co-prime, you may receive an incorrect result
```

Comment:
This warning still applies and shouldn't be deleted. Just modify the `crt([12, 6, 17], [3, 4, 2])` example to show the ValueError
```python
>>> crt([12, 6, 17], [3, 4, 2])
Traceback etc...
...
ValueError: etc...
```
i.e. the result will be 3 lines: the first and last lines of the error message with "..." as the 2nd line.

Reply:
Move your comment about the error to precede the example.

Reply:
But will it pass the doctest if I put something like above in the example of docstring? As the doctest carries test of its own

Reply:
I think @smichr means to replace the `etc...` parts with what is actually printed. The line with `...` will take care of everything between the first and last lines of the error output in the doctest runner.

Reply:
Ok @asmeurer got it. Thanks!